src/electionguard_tools/strategies/election.py

```diff
@@ -176,16 +176,6 @@ def reporting_unit_types(draw: _DrawType):
     return ReportingUnitType(n)
 
 
-@composite
-def annotated_emails(draw: _DrawType):
```

Comment:
deleting unused method?

Reply:
It's worse than that. It's a duplicated method. Exactly the same named method.